### PR TITLE
Add systemd script

### PR DESCRIPTION
**Pull request by tschuy**
This script was provided by Adam at GA Tech.

As he mentioned, it should be extended with Environment/EnvironmentFile, but I was unable to get this working no matter how hard I banged my head against the wall.

### CODE REVIEW

File: docs/source/getting_started/installing.rst [hunk 1]
```diff
@@ -92,6 +99,22 @@ The ``vncauthproxy`` service is installed and can be started::
 
     $ sudo service vncauthproxy start
 
+systemd
+~~~~~~~
+
+For systems running ``systemd``, a basic systemd script has been provided by Adam at
+the Georgia Tech Information Security Center. It has been tested on Debian 8.
```
Comment (ramereth) [inline docs/source/getting_started/installing.rst]: Has he given us permission to use his name in the docs?

File: docs/source/getting_started/installing.rst [hunk 2]
```diff
@@ -92,6 +99,22 @@ The ``vncauthproxy`` service is installed and can be started::
...
+
+Copy the file ``scripts/vncauthproxy/init-systemd`` to
+``/lib/systemd/system/vncauthproxy.service`` and enable the service::
+
+    $ sudo cp /path/to/gwm/scripts/vncauthproxy/init-systemd /lib/systemd/system/vncauthproxy.service
+    $ sudo systemctl enable vncauthproxy
+
+The script does not currently support variables, so if you want VNC AuthProxy
```
Comment (ramereth) [inline docs/source/getting_started/installing.rst]: Aside from the binary location needing to be absolute, were you not able to at least get variables working with the port/ip address to bind to? Ideally it should source a file in ``/etc/default/vncauthproxy``.

Comment (tschuy) [inline docs/source/getting_started/installing.rst]: I was not at all, which I found very odd. I'm probably doing something very wrong. 

Do we have a service that's customized that I could take a look at? Perhaps something managed by Chef I could just look at the chef code of? I'm probably using them wrong.

File: docs/source/getting_started/installing.rst (hunk 1 above)
Comment (tschuy) [inline docs/source/getting_started/installing.rst]: He has not. If he doesn't respond to me asking on the issue, what would the best way to go about citing it be? Link to the issue? Or consider it trivial enough and not cite it?

File: docs/source/getting_started/installing.rst (hunk 2 above)
Comment (jordane) [inline docs/source/getting_started/installing.rst]: I tested and as able to get this working. Ping me when you're in next and I can help you with it.

Comment (jordane) [inline docs/source/getting_started/installing.rst]: We talked about this in person, seems binary location must be absolute, other options are variable.

**Comment by Kennric** [comment]
+1

**Pull request merged**